src/include/sof/audio/asrc/asrc_farrow.h

```diff
@@ -136,8 +143,9 @@ enum asrc_error_code {
 						/*!< control mode. */
 	ASRC_EC_FAILED_PUSH_MODE = -13,		/*!< Push mode operation */
 						/*!< failed. */
-	ASRC_EC_FAILED_PULL_MODE = -14		/*!< Pull mode operation */
+	ASRC_EC_FAILED_PULL_MODE = -14,		/*!< Pull mode operation */
```

Comment:
Can you also make the comment be on one line? Remember, 100 chars limit.

Reply:
You were fast -- I accidentally  re-requested new review. Sure!

Reply:
Yeah I acted on the force-push notification and saw that I hadn't fully reviewed it yet anyway.

Reply:
Still missed this one in particular.

Reply:
This is fixed now, pressing resolved.